src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -52,29 +52,15 @@ public void onFinish(ITestContext context) {
         logger.info("The test class run finished on ... " + context.getEndDate());
     }
 
-    @Attachment(value = "Test.log", type = "text/plain")
-    public static byte[] appendLogToAllure() {
-        try {
-            return FileUtils.readFileToByteArray(new File("target/test.log"));
-        } catch (IOException ignored) {
-            return null;
-        }
-    }
-
-    public static void clearFile() throws IOException {
-        File myFoo = new File("target/test.log");
-        FileOutputStream fooStream = new FileOutputStream(myFoo, false);
-        byte[] myBytes = "".getBytes();
-        fooStream.write(myBytes);
-        fooStream.close();
-    }
-
     @Override
     public void onTestSuccess(ITestResult result) {
         logger.info(result.getName() + " successful...");
         try {
-            appendLogToAllure();
-            clearFile();
+            AllureUtil.appendLogToAllure();
+            LogUtil.clearFile();
```

Comment:
I think clearing file is part of FileUtils rather than LogUtil
BTW you can use log property to clear log file by default (log4j.appender.file.append)

Reply:
done